llpc/lower/llpcSpirvLowerGlobal.h

```diff
@@ -118,6 +123,8 @@ class SpirvLowerGlobal : public SpirvLower, public llvm::PassInfoMixin<SpirvLowe
   void interpolateInputElement(unsigned interpLoc, llvm::Value *interpInfo, llvm::CallInst &callInst,
                                GlobalVariable *gv, ArrayRef<Value *> indexOperands);
 
+  function_ref<DominatorTree &(Function &)> m_lookupDomTree;
```

Comment:
Now that we have dropped the old pass manager there is no need for this lookupDomTree abstraction. I will clean this up.